### DISCUSSION TITLE
如何在一个项目中同时依赖两个不同版本的antd

### DISCUSSION THREAD

**Discussion by ShuaiJunlan** [ant-design/ant-design#33300] [Q&A]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
比如说，项目原始依赖的版本是"antd": "3.5.0"，但是想用一些新组件然后需要依赖"antd": "3.26.19"，考虑到兼容性问题，不想直接升级，想同时依赖两个版本

### What does the proposed API look like?
null

**Comment by crazyair** [comment]
1. 模块联邦
2. external

**Reply by ShuaiJunlan** [reply]
有相关的参考文档么，我对这块不是很熟悉

**Comment by eightHundreds** [comment]
1. mobile的方案，"antd-mobile-v5": "npm:antd-mobile@5"
2. 样式污染问题靠改前缀
